sal_interfaces/ATDome/ATDome_Events.xml

```diff
@@ -245,6 +301,34 @@
             <Units>unitless</Units>
             <Count>1</Count>
         </item>
+        <item>
+            <EFDB_Name>coast</EFDB_Name>
+            <Description>Azimuth coasting distance.</Description>
+            <IDL_Type>float</IDL_Type>
+            <Units>deg</Units>
+            <Count>1</Count>
+        </item>
+        <item>
+            <EFDB_Name>encoderCountsPer360</EFDB_Name>
+            <Description>Number of azimuth encoder counts per full rotation.</Description>
+            <IDL_Type>long long</IDL_Type>
+            <Units>unitless</Units>
+            <Count>1</Count>
+        </item>
+        <item>
+            <EFDB_Name>azimuthMoveTimeout</EFDB_Name>
+            <Description>Timeout for azimuth moves.</Description>
+            <IDL_Type>float</IDL_Type>
+            <Units>second</Units>
+            <Count>1</Count>
+        </item>
+        <item>
+            <EFDB_Name>doorMoveTimeout</EFDB_Name>
+            <Description>Timeout for opening or closing (what? either shutter door?).</Description>
```

Comment:
This is how long it waits to fully close the shutters before timing out.

Reply:
Only close, not open?